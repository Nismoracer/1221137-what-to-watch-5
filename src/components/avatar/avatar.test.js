import React from "react";
import renderer from "react-test-renderer";
import Avatar from "./avatar";
import {BrowserRouter} from 'react-router-dom';

it(`Should Avatar render correctly`, () => {
  const tree = renderer
    .create(
        <BrowserRouter>
          <Avatar/>
        </BrowserRouter>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
