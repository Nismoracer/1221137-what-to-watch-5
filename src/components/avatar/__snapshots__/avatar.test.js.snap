// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Avatar render correctly 1`] = `
<a
  href="/mylist"
  onClick={[Function]}
>
  <div
    className="user-block__avatar"
  >
    <img
      alt="User avatar"
      height="63"
      src="img/avatar.jpg"
      width="63"
    />
  </div>
</a>
`;
